app/frontend/src/Settings/OverlaySettings.js

```diff
@@ -37,16 +37,24 @@ const OverlaySettings = () => {
               When a line is activated in the controller and seen on the presenter,
               the same information is used to generate the overlay.
               Choose what to display and how to display it with the options below.
-              Currently only one overlay configuration is available.
+              You can choose from a variety of pre-configured overlays or
+              create your own using theme tool.
```

Comment:
```suggestion
              create your own using the theme tool.
```

But try to make sure the text doesn't overflow to another line if possible.